test/gtest/ucp/test_ucp_tag_xfer.cc

```diff
@@ -157,14 +157,16 @@ void test_ucp_tag_xfer::test_xfer_generic(size_t size, bool expected, bool sync)
 
 void test_ucp_tag_xfer::test_xfer_iov(size_t size, bool expected, bool sync)
 {
+    const size_t iovcnt = 20;
     std::vector<char> sendbuf(size, 0);
     std::vector<char> recvbuf(size, 0);
 
     ucs::fill_random(sendbuf.begin(), sendbuf.end());
 
-    UCS_TEST_GET_BUFFER_DT_IOV(iov, iovcnt, sendbuf.data(), sendbuf.size(), 20);
+    UCS_TEST_GET_BUFFER_DT_IOV(send_iov, send_iovcnt, sendbuf.data(), sendbuf.size(), iovcnt);
+    UCS_TEST_GET_BUFFER_DT_IOV(recv_iov, recv_iovcnt, recvbuf.data(), recvbuf.size(), iovcnt);
 
-    size_t recvd = do_xfer(&iov, &recvbuf[0], iovcnt, DATATYPE_IOV, DATATYPE_IOV,
```

Comment:
@shamisp Minor conflict in `test/gtest/ucp/test_ucp_tag_xfer.cc` has been resolved.
Strictly speaking, it is not clear how it could work - `&recvbuf[0]` was not a `DATATYPE_IOV`
Anyway, currently I expect everything should work well.